test/base/src/androidTest/kotlin/io/realm/kotlin/test/RealmTests.kt

```diff
@@ -26,9 +26,7 @@ import kotlin.test.BeforeTest
 import kotlin.test.Test
 import kotlin.test.assertFailsWith
 import kotlin.test.assertFalse
-import kotlin.time.ExperimentalTime
 
-@OptIn(ExperimentalTime::class)
```

Comment:
Is this the only ocurrence of `ExperimentalTime`?

Reply:
I could remove it here, because we only use Duration which is stable. Other parts of the Kotlin Time API still needs this annotation.